Libplanet/Net/Messages/Message.cs

```diff
@@ -140,16 +149,58 @@ private enum MessageFrame
             Sign = 3,
         }
 
+        /// <summary>
+        /// Identity bytearray of the message.
+        /// If a message B is the reply of the message A,
+        /// B's identity must be set to A's identity.
+        /// </summary>
         public byte[] Identity { get; set; }
 
+        /// <summary>
+        /// <see cref="AppProtocolVersion"/>-typed version of the
+        /// <see cref="Remote"/>'s transport layer.
+        /// </summary>
         public AppProtocolVersion Version { get; set; }
 
+        /// <summary>
+        /// Sender <see cref="Peer"/> of the message.
+        /// </summary>
         public Peer Remote { get; set; }
 
         protected abstract MessageType Type { get; }
 
         protected abstract IEnumerable<NetMQFrame> DataFrames { get; }
 
+        /// <summary>
+        /// Cast given <see cref="ToNetMQMessage"/>-typed <paramref name="raw"/> to
```

Comment:
I wonder that it is a correct usage of *-typed*. In my thought, it should be `NetMQMessage` instead.